src/server/services/tests/TEST_MEDIA_LINKS.txt

```diff
@@ -0,0 +1,11 @@
+	Updated:	7/24/20	3:36pm								

```

Comment:
@maxpluez I see this file has some data errors. So it should be named BAD_TEST_MEDIA_LINKS.txt, then need a corresponding test for GOOD_TEST_MEDIA_LINKS.txt

Also, in unit tests a file like this is called a fixture. So put this in your test folder under a folder called "fixtures"